lib/common/transforms.js

```diff
@@ -696,6 +696,40 @@ module.exports = {
     }
   },
 
+  /**
+   * Scales the pixel value to rem, and adds 'rem' to the end of non-zero values. If you define a "basePxFontSize" on the platform in your config, it will be used to scale the pixel value, otherwise 16 (default web font size) will be used.
+   *
+   * ```js
+   * // Matches: prop.attributes.category === 'size'
+   * // Returns:
+   * "0"
+   * "1rem"
+   * ```
+   */
+  'size/pxToRem': {
+    type: 'value',
+    matcher: (prop) => prop.attributes.category === 'size',
```

Comment:
This could use the `isSize` function defined at the top of this file for brevity and DRY-ness: `matcher: isSize,`

Reply:
🤦 good call